cliquet/resource.py

```diff
@@ -282,7 +290,8 @@ def collection_delete(self):
         filters = self._extract_filters()
         deleted = self.collection.delete_records(filters=filters)
 
-        return self.postprocess(deleted)
+        action = len(deleted) > 0 and ACTION_DELETE or ACTION_READ
```

Comment:
I agree with tarek here, let's start with something straightforward, and let the listeners do the filtering.

In the future we can imagine to introduce what you suggest from the configuration (per resource, per action, etc.)